developer/src/server/build.sh

```diff
@@ -129,17 +129,19 @@ if (( build_keymanweb )); then
 fi
 
 if (( copy_keymanweb )); then
-  SRC="$KEYMAN_ROOT/web/release/unminified"
+  WEB_SRC="$KEYMAN_ROOT/build/web/debug"
+  UI_SRC="$KEYMAN_ROOT/build/ui/debug"
```

Comment:
Current Developer build failed because I forgot an important part of the path:

```suggestion
  WEB_SRC="$KEYMAN_ROOT/web/build/web/debug"
  UI_SRC="$KEYMAN_ROOT/web/build/ui/debug"
```